rosbag2_cpp/include/rosbag2_cpp/cache/message_cache.hpp

```diff
@@ -71,18 +71,21 @@ class ROSBAG2_CPP_PUBLIC MessageCache
 public:
   explicit MessageCache(size_t max_buffer_size);
 
-  ~MessageCache();
+  ~MessageCache() override;
 
   /// Puts msg into primary buffer. With full cache, msg is ignored and counted as lost
   void push(std::shared_ptr<const rosbag2_storage::SerializedBagMessage> msg) override;
 
   /// Gets a consumer buffer.
   /// In this greedy implementation, swap buffers before providing the buffer.
-  std::shared_ptr<CacheBufferInterface> consumer_buffer() override;
+  std::shared_ptr<CacheBufferInterface> get_consumer_buffer() override;
 
-  /// Notify that consumer_buffer has been fully used. Unlock.
+  /// Notify that get_consumer_buffer has been fully used. Unlock.
```

Comment:
Ohh, sorry for the confusion. It's messed up a little bit after renaming method `consumer_buffer` to the `get_consumer_buffer`. I will change it.
I am going to change it to the:
`/// Signals that tne consumer is done consuming, unlocking the buffer so it may be swapped.`

Please let me know if you have any concerns about wording.

Reply:
That looks like a great change to me! Nice and clear!

Reply:
Fixed.